src/components/chat/w2wChat/groupChat/groupInfo/groupInfoModalContent.tsx

```diff
@@ -269,192 +285,294 @@ export const GroupInfoModalContent = ({ onClose }: ModalInnerComponentType) => {
+                margin="0px 0px 21px 0px"
+              >
+                <Lock />
+                <ItemVV2
+                  alignItems="flex-start"
+                  margin="0px 0px 0px 12px"
+                >
+                  <SpanV2
+                    fontSize="18px"
+                    fontWeight="500"
+                    color={theme.default.color}
+                  >
+                    {currentChat?.groupInformation?.isPublic ? 'Public' : 'Private'}
+                  </SpanV2>
+                  <SpanV2
+                    fontSize="12px"
+                    fontWeight="400"
+                    color={theme.modalIconColor}
+                  >
+                    {currentChat?.groupInformation?.isPublic ? 'Chats are not encrypted' : 'Chats are encrypted'}
+                  </SpanV2>
+                </ItemVV2>
+              </InfoContainer>
+              {isAccountOwnerAdmin && currentChat?.groupInformation?.members?.length < 10 && (
+                <AddWalletContainer onClick={() => setShowAddMoreWalletModal(true)}>
+                  <AddMember />
+                  <SpanV2
+                    color={theme.modalProfileTextColor}
+                    margin="0px  14px"
+                    fontSize="18px"
+                    fontWeight="400"
+                  >
+                    Add more wallets
+                  </SpanV2>
+                </AddWalletContainer>
+              )}
+              {currentChat?.groupInformation?.pendingMembers?.length > 0 && (
+                <ItemVV2
+                  border={`1px solid ${theme.default.border}`}
+                  borderRadius="16px"
+                  margin="0px 0px 15px 0px"
+                  maxWidth="445px"
+                  style={{ boxSizing: 'border-box' }}
+                >
+                  <ItemHV2
+                    justifyContent="space-between"
+                    padding="13px 20px 13px 16px"
+                    onClick={() => setshowPendingRequests(!showPendingRequests)}
+                  >
+                    <ItemHV2 justifyContent="flex-start">
+                      <SpanV2
+                        fontSize="18px"
+                        fontWeight="400"
+                        color={theme.modalProfileTextColor}
+                      >
+                        Pending Requests
+                      </SpanV2>
+                      <SpanV2
+                        background="#CF1C84"
+                        color="#FFFFFF"
+                        fontSize="13px"
+                        fontWeight="700"
+                        borderRadius="8px"
+                        padding="4px 12px"
+                        margin="0px 0px 0px 8px"
+                      >
+                        {currentChat?.groupInformation?.pendingMembers?.length}
+                      </SpanV2>
+                    </ItemHV2>
+                    {showPendingRequests ? <DropdownUp /> : <Dropdown />}
```

Comment:
DropdownUp icon is already present in Profile component in src/primaries, check the implementation there
<img width="662" alt="image" src="https://user-images.githubusercontent.com/42746736/225235294-123d0e5b-4e40-4661-b225-b7a1cde258e6.png">